cert_issuer/blockchain_handlers/bitcoin/__init__.py

```diff
@@ -33,9 +32,9 @@ def get_recommended_fee_coin(self):
 def initialize_signer(app_config):
     path_to_secret = os.path.join(app_config.usb_name, app_config.key_file)
 
-    if app_config.chain.blockchain_type == BlockchainType.bitcoin:
+    if app_config.chain.is_bitcoin_type():
```

Comment:
I cannot see where this method is defined?

Reply:
This is a new method on this [cert-core PR](https://github.com/blockchain-certificates/cert-core/pull/4).
Before this refactoring commit, this PR needed `cert-core` changes to define new Ethereum testnets, so I added additional changes to add useful methods like `is_bitcoin_type ` on the Chain class.